src/javascript/app/pages/user/account/authenticate.js

```diff
@@ -117,6 +122,24 @@ const Authenticate = (() => {
         }
     };
 
+    const initEdd = () => {
+        file_checks_edd    = {};
+        $submit_status_edd = $('.submit-status-edd');
+        $submit_table_edd  = $submit_status_edd.find('table tbody');
+
+        // Setup accordion
+        $('#not_authenticated_edd .files').accordion({
```

Comment:
`not_authenticated_edd ` is being used in multiple places, it is best to define it once and reuse the variable name.